### ISSUE TITLE
NoClassDefFoundError: com.mixpanel.android.mpmetrics.MixpanelActivityLifecycleCallbacks

### ISSUE THREAD

**Issue by ismaeltoe** [mixpanel/mixpanel-android#374]
My app is crashing for some users. I don't know if it's always or sometimes. All devices on which it crashes run Android 2.3.3 to 2.3.7. I'm using Mixpanel 4.8.7.

STACK TRACES:

```
java.lang.NoClassDefFoundError: com.mixpanel.android.mpmetrics.MixpanelActivityLifecycleCallbacks
	at com.mixpanel.android.mpmetrics.MixpanelAPI.registerMixpanelActivityLifecycleCallbacks(MixpanelAPI.java:1271)
	at com.mixpanel.android.mpmetrics.MixpanelAPI.<init>(MixpanelAPI.java:246)
	at com.mixpanel.android.mpmetrics.MixpanelAPI.<init>(MixpanelAPI.java:196)
	at com.mixpanel.android.mpmetrics.MixpanelAPI.getInstance(MixpanelAPI.java:300)
	at com.tomsyweb.soc.MainActivity.onCreate(MainActivity.java:35)
```
The line 35 of MainActivity.java:

`mMixpanel = MixpanelAPI.getInstance(this, MIXPANEL_API_TOKEN);`

The devices on which it crashes, if it can help (I don't know if those are the only but it's those that appear in my developer console):

- Galaxy S Glide
- Xperia U
- Galaxy Pocket.

**Comment by patedit** [comment]
@ismaeltoe This issue will be resolved with 4.8.8 which is coming out by the end of this week. In the meanwhile, you can always point your `gradle` file to use `master` branch from this repo. I'm so sorry for the inconvenience!

**Comment by ismaeltoe** [comment]
Hi, is it resolved in 4.9?

**Comment by patedit** [comment]
It is resolved in the latest release, yes 👍
